src/diffusers/loaders/peft.py

```diff
@@ -31,6 +31,7 @@
 _SET_ADAPTER_SCALE_FN_MAPPING = {
     "UNet2DConditionModel": _maybe_expand_lora_scales,
     "SD3Transformer2DModel": lambda model_cls, weights: weights,
+    "UNetMotionModel": lambda model_cls, weights: weights,
```

Comment:
Pull from upstream once https://github.com/huggingface/diffusers/pull/8995 is merged.